crates/re_time_panel/src/lib.rs

```diff
@@ -417,13 +417,10 @@ impl TimePanel {
         tree: &EntityTree,
         ui: &mut egui::Ui,
     ) {
-        if !tree
+        let tree_has_data_in_current_timeline = tree
             .prefix_times
             .has_timeline(ctx.rec_cfg.time_ctrl.timeline())
-            && tree.num_timeless_messages() == 0
-        {
-            return; // ignore entities that have no data for the current timeline, nor any timeless data.
-        }
+            || tree.num_timeless_messages() > 0;
```

Comment:
It's great that you made a readable variable out of this, but let's take it to the logical conclusion and make helper function for this: `tree.has_data_in_current_timeline` or similar

Reply:
Also note that this will return true for `world` if data is logged to `world/point`, since `world` is a prefix of `world/point`

Reply:
> Also note that this will return true for world if data is logged to world/point, since world is a prefix of world/point

Yeah, that's the desired behaviour for reporting "emptiness" I think.